ui/src/components/common/Header.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  Page,
+  PageHeader,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarItem
+} from '@patternfly/react-core';
+import { UserIcon } from '@patternfly/react-icons';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { css } from '@patternfly/react-styles';
+import './Header.css';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { toggleHeaderDropdown } from '../../actions/actions-ui';
+
+class Header extends React.Component {
+  onTitleClick = () => {};
+
+  onUserDropdownToggle = () => {
```

Comment:
@damienomurchu Looks like this method is a duplicate of `onUserDropdownSelect()`

Reply:
@laurafitzgerald both needed - `onUserDropdownToggle` will be triggered when the unexpanded dropdown is clicked on, and `onUserDropdownSelect` is needed to close the dropdown when a dropdown item has been selected